index.html

```diff
@@ -1,140 +1,159 @@
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" 
-    integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" 
-    crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+        crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+        type="text/css" />
     <title>Microverse Capstone One</title>
 </head>
-<body>
-    <div>
-        <header class="header-menu">
-            <nav>
-                <a href="" class="mx-3 text-white"><i class="fab fa-facebook"></i></a>
-                <a href="" class="mx-3 text-white"><i class="fab fa-twitter"></i></a>
-                <a href="" class="mx-3 text-white">English</a>
-                <a href="" class="mx-3 text-white">MyPage</a>
-                <a href="" class="mx-3 text-white">Logout</a>
 
-            </nav>
+<body>
+    <header>
+        <nav class="header-menu">
+            <a href="" class="mx-3 text-white"><i class="fab fa-facebook"></i></a>
+            <a href="" class="mx-3 text-white"><i class="fab fa-twitter"></i></a>
+            <a href="" class="mx-3 text-white">English</a>
+            <a href="" class="mx-3 text-white">MyPage</a>
+            <a href="" class="mx-3 text-white">Logout</a>
+        </nav>
         <div class="desktop-menu">
-            <div>
-                <div>
-                    <a href="index.html">
-                        <span>Open Source Global</span>
-                        <span>Submit 2021</span>
-                    </a>
-                </div>
-                <div>
-                    <a href="about.html">About</a>
-                    <a href="">Program</a>
-                    <a href="">Join</a>
-                    <a href="">Sponsor</a>
-                    <a href="">News</a>
-                    <a href="">OS Campaign</a>
-                </div>
+            <div class="desktop-container">
+                <a class="os-logo" href="index.html">
+                    <i class="fab fa-osi"></i>
+                    <span>Open Source Global<br></span>
+                    <span>Submit 2021</span>
+                </a>
+                <div class="menu-links">
+                <a class="about" href="about.html">About</a>
+                <a href="">Program</a>
+                <a href="">Join</a>
+                <a href="">Sponsor</a>
+                <a href="">News</a>
+                <a class="OS-campaign" href="">OS Campaign</a>
+            </div>
```

Comment:
As required by the previous code reviewer, Let's make your webpage header more identical to the design guidelines 👍 

> Your design header
![Screenshot (316)](https://user-images.githubusercontent.com/66112760/142592691-a321b25b-f905-4734-905b-c030fdcc9c95.png)

> Design header
![Screenshot (317)](https://user-images.githubusercontent.com/66112760/142592776-7925a3e9-0e81-4c11-9291-ec9b2e330d4f.png)